client/src/pages/blocked.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Grid, Col, Row } from '@freecodecamp/react-bootstrap';
+import Helmet from 'react-helmet';
+
+import { Spacer } from '../components/helpers';
+import './blocked.css';
+
+function BlockedPage(): JSX.Element {
+  return (
+    <>
+      <Helmet title={`Access Denied | freeCodeCamp.org`} />
+      <Grid className='text-center'>
+        <Spacer size={2} />
+        <Row>
+          <Col lg={8} lgOffset={2} sm={10} smOffset={1} xs={12}>
+            <h1>Your login has been blocked</h1>
+            <Spacer size={2} />
+            <Col lg={10} lgOffset={1} sm={10} smOffset={1} xs={12}>
+              <div className='text-left blocked-body'>
+                <p>
+                  To comply with U.S. export control and economic sanctions laws
+                  and regulations, as well as with our partners, you can not log
+                  in to freeCodeCamp from your region.
+                </p>
+                <p>
+                  However, you can continue to access thousands of hours of
+                  curriculum and other free resources without an account.
+                </p>
+                <p>
+                  Don&apos;t hesitate to{' '}
+                  <a href='https://www.freecodecamp.org/support'>contact us</a>{' '}
```

Comment:
Two thoughts:
1) There is a lot of unrelated info on this page (article), and the only related piece is "Go to the forum to ask a question". Is this article being updated?
2) If we are sending Campers to the forum, let us remember to give mods context for this kind of event/question.

Reply:
Yeah - the copy is not final. Just something I threw together to get the page going. 

It is being proofread by Quincy, and we will have a slightly different copy here. I would not worry about it just yet.

Reply:
Copy has been updated.